docs/public/static/manifest.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "MUI",
   "short_name": "MUI",
-  "display": "minimal-ui",
+  "display": "MUI",
```

Comment:
What was minimal-ui? 😄

Reply:
😂 mystery

Reply:
It's not a typo: https://developer.mozilla.org/en-US/docs/Web/Manifest/display#values

Reply:
TIL, thanks Olivier :))